javascript/modules/client/src/client-multisig.ts

```diff
@@ -0,0 +1,353 @@
+import { Random } from "@aragon/sdk-common";
```

Comment:
We should not be having client-multisig and client-allowlist... if they are doing the same